Makefile

```diff
@@ -50,6 +52,13 @@ upload-deskpro-secrets: check-env-vars ## Decrypt and upload DeskPro credentials
 	$(if $(wildcard ${DESKPRO_PASSWORD_STORE_DIR}),,$(error Password store ${DESKPRO_PASSWORD_STORE_DIR} does not exist))
 	@scripts/upload-deskpro-secrets.sh
 
+.PHONY: upload-zendesk-secrets
+upload-zendesk-secrets: check-env-vars ## Decrypt and upload DeskPro credentials to S3
+	$(eval export ZENDESK_PASSWORD_STORE_DIR?=${HOME}/.paas-pass)
+	$(if ${ZENDESK_PASSWORD_STORE_DIR},,$(error Must pass ZENDESK_PASSWORD_STORE_DIR=<path_to_password_store>))
+	$(if $(wildcard ${ZENDESK_PASSWORD_STORE_DIR}),,$(error Password store ${ZENDESK_PASSWORD_STORE_DIR} does not exist))
+	@scripts/upload-zendesk-secrets.sh
```

Comment:
Instead of adding a new task here, could we not rename the existing one to something like `upload-product-page-secrets` and include both in there?

That would follow the pattern used for rubbernecker which has secrets for talking to more than one remote thing.

Reply:
But then we duplicate secrets.